<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>info_user</class>
 <widget class="QMainWindow" name="info_user">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget { background: rgb(243, 246, 255);color: rgb(8, 104, 173);selection-color:#0078d7}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>17</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(51, 51, 51);</string>
      </property>
      <property name="text">
       <string>Thông tin</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="anh_dai_dien">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>200</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>15</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Pic 1*1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;
border-radius: 0px;
color: rgb(51, 51, 51);</string>
            </property>
            <property name="text">
             <string>FIle ảnh định dạng JPG, JPEG, PNG</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">border: 0px;
color: rgb(51, 51, 51);
border-radius: 0px;</string>
            </property>
            <property name="text">
             <string>Kích thước không quá 20MB</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>10</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_30">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <family>Segoe UI</family>
                <pointsize>12</pointsize>
                <italic>false</italic>
                <bold>false</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
</string>
              </property>
              <property name="text">
               <string>Tên đăng nhập</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ten_dang_nhap">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_37">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
</string>
              </property>
              <property name="text">
               <string>Họ và tên</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="ho_va_ten">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_31">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
</string>
              </property>
              <property name="text">
               <string>Email</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="email">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_33">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
</string>
              </property>
              <property name="text">
               <string>Chức vụ</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="chuc_vu">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLabel" name="label_39">
              <property name="minimumSize">
               <size>
                <width>131</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
</string>
              </property>
              <property name="text">
               <string>Số sách mượn</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="so_sach_muon">
              <property name="styleSheet">
               <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,0,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label_42">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Thông tin cơ bản</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <property name="spacing">
             <number>10</number>
            </property>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label_47">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Ngày sinh</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_46">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Giới tính</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_48">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);</string>
                </property>
                <property name="text">
                 <string>Số điện thoại</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <widget class="QDateEdit" name="ngay_sinh">
                  <property name="styleSheet">
                   <string notr="true">/* --- PHẦN 1: Ô NHẬP LIỆU (Giống ComboBox) --- */
QDateEdit {
    border: 2px solid transparent;
    border-radius: 8px;
    padding: 6px 12px;
    background-color: #F3F4F6; /* Nền xám nhạt */
    color: #333;
    selection-background-color: rgb(69, 104, 173);
}

/* Khi di chuột vào */
QDateEdit:hover {
    background-color: #E5E7EB;
}

/* Khi đang nhập liệu (Focus) */
QDateEdit:focus {
    background-color: white;
    border: 2px solid rgb(69, 104, 173); /* Viền xanh */
}

/* Nút mũi tên xổ xuống */
QDateEdit::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

/* Icon (Bạn có thể dùng icon lịch 'calendar.png' thay vì mũi tên cho đúng điệu) */
QDateEdit::down-arrow {
    image: url(:/icons/icons_/down-chevron.png); /* Hoặc file icon lịch của bạn */
    width: 16px;
    height: 16px;
}

/* --- PHẦN 2: LỊCH XỔ XUỐNG (QCalendarWidget) --- */

/* Khung tổng thể của lịch */
QCalendarWidget QWidget {
    alternate-background-color: #f9f9f9; 
    background-color: white;
    border-bottom-right-radius: 8px;
    border-bottom-left-radius: 8px;
}

/* Thanh điều hướng phía trên (Tháng/Năm) */
QCalendarWidget QToolButton {
    color: #333;
    background-color: transparent;
    border: none;
    font-weight: bold;
    icon-size: 20px; /* Chỉnh kích thước mũi tên trái phải của lịch */
}
QCalendarWidget QToolButton:hover {
    background-color: #F3F4F6;
    border-radius: 4px;
}

/* Menu chọn tháng năm khi bấm vào tiêu đề */
QCalendarWidget QMenu {
    background-color: white;
    border: 1px solid #ccc;
}

/* Bảng ngày tháng (Spin view) */
QCalendarWidget QAbstractItemView:enabled {
    font-size: 13px;
    color: #333;
    background-color: white;
    selection-background-color: rgb(69, 104, 173); /* Màu nền ngày được chọn */
    selection-color: white; /* Màu chữ ngày được chọn */
    outline: 0; /* Bỏ đường viền nét đứt xấu xí */
}

/* Ngày trong tuần (T2, T3, T4...) */
QCalendarWidget QAbstractItemView:disabled {
    color: #888; /* Màu chữ xám cho ngày không thuộc tháng hiện tại */
}</string>
                  </property>
                  <property name="currentSection">
                   <enum>QDateTimeEdit::Section::DaySection</enum>
                  </property>
                  <property name="displayFormat">
                   <string>d/M/yyyy</string>
                  </property>
                  <property name="calendarPopup">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="gioi_tinh">
                  <property name="styleSheet">
                   <string notr="true">QComboBox {
    background-color: #f3f4f6;
    border: 1px solid #f3f4f6;
    border-radius: 8px;
    padding: 6px 12px;
    color: #333;
}

QComboBox:hover {
    background-color: #e4e6eb;
    border-color: #e4e6eb;
}

QComboBox:on {
    background-color: #e4e6eb;
    border-bottom-left-radius: 0px;
    border-bottom-right-radius: 0px;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left-width: 0px;
    border-top-right-radius: 8px;
    border-bottom-right-radius: 8px;
}

QComboBox::down-arrow {
    image: url(:/icons/icons_/down-chevron.png);
    width: 12px;
    height: 12px;
}

QComboBox QAbstractItemView {
    background-color: white;
    border: 1px solid #dcdcdc;
    border-radius: 8px;
    border-top-left-radius: 0px;
    border-top-right-radius: 0px;
    color: #333;
    selection-background-color: #f3f4f6;
    selection-color: #333;
    outline: 0;
}

QComboBox QAbstractItemView::item {
    padding: 8px 12px;
    min-height: 25px;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #f0f5ff;
    color: rgb(60, 91, 149);
}

QComboBox QAbstractItemView::item:selected {
    background-color: #e4e6eb;
    color: #333;
}</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>Nam</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Nữ</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Khác</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="so_dien_thoai">
                  <property name="styleSheet">
                   <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_41">
            <property name="font">
             <font>
              <pointsize>13</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(0, 0, 0);</string>
            </property>
            <property name="text">
             <string>Thực hiện thay đổi mật khẩu</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QLabel" name="label_35">
                <property name="font">
                 <font/>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
</string>
                </property>
                <property name="text">
                 <string>Mật khẩu mới</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_36">
                <property name="styleSheet">
                 <string notr="true">color: rgb(0, 0, 0);
</string>
                </property>
                <property name="text">
                 <string>Xác nhận mật khẩu mới</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QLineEdit" name="mat_khau_moi">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="mat_khau_moi_again">
                <property name="styleSheet">
                 <string notr="true">QLineEdit {
    background-color: #e8ecef; 
    border-radius: 5px; 
    border: 1px solid #E0E4E8; 
    padding: 8px 10px; 
    color: #1E1E1E; 
    font-size: 14px;
}

QLineEdit:hover {
    background-color: #E5E9ED;
}
QLineEdit:focus {
    background-color: #FFFFFF; 
    border: 1px solid #3498db; 
}</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2ecc71; /* Xanh lá tươi (Emerald) */
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #58d68d; /* Sáng hơn khi di chuột */
}

QPushButton:pressed {
    background-color: #27ae60; /* Xanh đậm (Nephritis) khi ấn */
    padding-top: 11px;
}</string>
          </property>
          <property name="text">
           <string>Mở khóa tài khoản</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #e74c3c; /* Đỏ tươi (Alizarin) - Thay cho xanh dương */
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #ff6b6b; /* Đỏ nhạt/sáng hơn khi di chuột */
}

QPushButton:pressed {
    background-color: #c0392b; /* Đỏ đậm (màu gạch) khi ấn xuống */
    padding-top: 11px;
}</string>
          </property>
          <property name="text">
           <string>Khóa tài khoản</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="luu_thong_tin">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #3498db; /* Xanh dương tươi (Bright Blue) */
    color: white;
    border: none;
    border-radius: 8px;
    padding: 10px 20px;
    font-weight: bold;
    font-size: 14px;
}

QPushButton:hover {
    background-color: #5dade2; /* Sáng hơn khi di chuột */
}

QPushButton:pressed {
    background-color: #2980b9; /* Tối hơn chút khi ấn */
    padding-top: 11px;
}</string>
          </property>
          <property name="text">
           <string>Lưu thay đổi</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
